map2csv/webtool_map2csv.py

```
@@ -0,0 +1,238 @@
+        next_idx = len(st.session_state["coords"]) + 1
+        st.write(f"### 画像上で対応点をクリックしてください。2つの点はできるだけ離れた位置にしてください")
+        # 2点以上で2Dアフィン変換を計算する前のみ案内文を表示
+        if len(st.session_state["coords"]) < 2 or len(st.session_state["latlons"]) < 2:
+            st.info(f"{next_idx}番目の点をクリックしてください")
+        # 印付き画像を作成
+        img_marked = img.copy()
+        draw = ImageDraw.Draw(img_marked)
+        font = None
+        try:
+            font = ImageFont.truetype("DejaVuSans-Bold.ttf", 24)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)